### PR TITLE
Update Main Class to deal with basic adding and deletion of tasks

### PR DESCRIPTION
**Pull request by Aak242**

### CODE REVIEW

File: src/main/java/seedu/duke/Duke.java [hunk 1]
```diff
@@ -2,20 +2,119 @@
```
Comment (Mahesh1772) [inline src/main/java/seedu/duke/Duke.java:2]: Good job not creating a arrowhead within the code!

File: src/main/java/seedu/duke/Duke.java [hunk 2]
```diff
@@ -2,20 +2,119 @@
...
+                System.out.println("Exiting Calendar...");
+                System.exit(0);
+                break;
+            default:
+                System.out.println("Invalid input. Please try again.");
+            }
+        }
+    }
+
+    private static void addTask(Scanner scanner, WeekView weekView, TaskManager taskManager) {
```
Comment (Mahesh1772) [inline src/main/java/seedu/duke/Duke.java:61]: This could fit better inside the TaskManager!

File: src/main/java/seedu/duke/Duke.java [hunk 3]
```diff
@@ -2,20 +2,119 @@
...
+        int taskNumber;
+        try {
+            taskNumber = Integer.parseInt(scanner.nextLine().trim());
+            taskManager.deleteTask(date, taskNumber - 1);
+            System.out.println("Task deleted.");
+        } catch (NumberFormatException e) {
+            System.out.println("Invalid task number. Please try again.");
+        }
+
+     }
```
Comment (clarencepohh) [inline src/main/java/seedu/duke/Duke.java:119]: As discussed in tutorial, I have already placed the addTask and deleteTask inside the TaskManager class. Let's fix this together in the merge.

Comment (NicholasTanYY) [inline src/main/java/seedu/duke/Duke.java:119]: This is good code, but as discussed previously, this falls under the task manager instead!

**Review by Mahesh1772 — COMMENTED** [review]
Refactor would be preferred

File: src/main/java/seedu/duke/Duke.java [hunk 4]
```diff
@@ -2,20 +2,119 @@
...
+                break;
+            case "delete":
+                deleteTask(scanner, weekView, taskManager);
+                break;
+            case "quit":
+                System.out.println("Exiting Calendar...");
+                System.exit(0);
+                break;
+            default:
+                System.out.println("Invalid input. Please try again.");
```
Comment (NicholasTanYY) [inline src/main/java/seedu/duke/Duke.java:56]: Great indentation, following the coding standards!

**Review by clarencepohh — COMMENTED** [review]
LGTM overall! Looks good as a base 'main()' function.

**Review by NicholasTanYY — COMMENTED** [review]
Overall, this is good code but remember to place code in the right places! Good job in maintaining the code quality!

**Pull request merged**